bee-network/src/service/host.rs

```diff
@@ -195,7 +204,7 @@ async fn peerstate_checker(shutdown: Shutdown, senders: Senders, peerlist: PeerL
     // We want to reduce the overhead of simultaneous mutual dialing even if several nodes are started at the same time
     // (by script for example). We do this here by adding a small random delay to when this task will be executing
     // regular peer state checks.
-    let delay = Duration::from_millis(rand::thread_rng().gen_range(0u64..1000));
+    let delay = Duration::from_millis(rand::thread_rng().gen_range(0u64..MAX_PEER_STATE_CHECKER_DELAY_MSECS));
```

Comment:
Kind of nitpicky, but I would change this to `..._MILLIS`, since `..._MSECS` could (potentially) refer to microseconds.

Reply:
oh you're right. Will change!